hasher-matcher-actioner/hmalib/common/label_models.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
Please change the name of this module back to classification_models.py. We'll be making different classes for the things that aren't classifications (like action label and writeback label)